File: alternate_vowels.js
function isAlt(word) {
    let vowels = ['a', 'e', 'i', 'o', 'u']
    for(j=1; j<word.length; j++){
      if((vowels.includes(word[j]) && vowels.includes(word[j-1])) || (!vowels.includes(word[j]) && !vowels.includes(word[j-1]))){
        return false
      }
    }
    return true
  }

console.log(isAlt("amazon"))

File: alternate_vowels.py
def is_alt(s):
    vowels = ['a', 'e', 'i', 'o', 'u']
    extractedVowels = []
    for j in range(1, len(s)):
        if (s[j] in vowels and s[j-1] in vowels) or (s[j] not in vowels and s[j-1] not in vowels):
            return False
    return True

print(is_alt("yay"))